frontend/src/lib/components/entityCard/EntityCard.svelte

```diff
@@ -154,8 +155,6 @@ In nested cards, the layout and rendering of contents varies from that of a pare
             {#if isCandidate(entity)}
               {#if nominatingParty}
                 <PartyTag party={nominatingParty} variant="short" />
-              {:else}
```

Comment:
We don't want to show 'unaffiliated' because we do not know whether the election has no parties at all or just that this candidate does not belong to one. The logic can be made robust with `vaa-data` later.